src/app/features/issue/providers/azuredevops/azuredevops-issue/azuredevops-issue-content/azuredevops-issue-content.component.scss

```diff
@@ -0,0 +1,56 @@
+@import 'src/variables';
+
+// table styled by ./src/styles/components/issue-table.scss
```

Comment:
random thought: would not an `@import ./src/styles/components/issue-table.scss` be possible instead of coping?

Reply:
TBH, I am not familiar with SCSS. I have copied from another issue module and replaced names 😅

Reply:
Well then you could just try that out, could not you? That `@import` is actually part of CSS (now), and it works quite simple: https://developer.mozilla.org/en-US/docs/Web/CSS/@import